cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/NoopScheduler.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.internal.app.runtime.schedule;
+
+import co.cask.cdap.api.schedule.SchedulableProgramType;
+import co.cask.cdap.api.schedule.Schedule;
+import co.cask.cdap.common.NotFoundException;
+import co.cask.cdap.proto.ScheduledRuntime;
+import co.cask.cdap.proto.id.NamespaceId;
+import co.cask.cdap.proto.id.ProgramId;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Noop scheduler.
+ */
+public class NoopScheduler implements Scheduler {
```

Comment:
Do not quite understand the usage of this class. Does this mean that we do not support schedule in preview mode and any request to the scheduler will be ignored?